java/Dockerfile

```diff
@@ -15,16 +15,11 @@
 # specific language governing permissions and limitations
 # under the License.
 
-FROM maven
+FROM maven:3.6-jdk-8
```

Comment:
Using `FROM maven:3.5.2-jdk-8-slim` allows me to build and run tests, can you use this base image?  With `maven:3.6-jdk-8` I get these errors when running the docker image:

```
[ERROR] org.apache.arrow.vector.TestVectorUnloadLoad  Time elapsed: 0.009 s  <<< ERROR!
java.lang.NoClassDefFoundError: org.junit.runner.Runner
	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:362)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:272)
	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:236)
	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:159)
```

Reply:
That was the solution, thanks!